examples/example_detector_config_files/public_ivc_config_SigGen.config

```diff
@@ -0,0 +1,51 @@
+# Example configuration file for fieldgen/siggen, by David Radford
```

Comment:
This is not an IVC detector. It's an PPC.
I guess you did this for the plot recipes, but I think we should name this correctly.
We should actually improve the general plot recipes instead.

Reply:
Good point. II've changed everthing including the filename, example link and runtest to ppc.